src/opnsense/mvc/app/controllers/OPNsense/Routing/forms/dialogEditGateway.xml

```diff
@@ -63,6 +63,13 @@
         <style>monitor_opt</style>
         <help>When a monitor down event is triggered, kill all states to this gateway.</help>
     </field>
+    <field>
+        <id>gateway_item.monitor_killstates_priority</id>
+        <label>Kill states metered</label>
```

Comment:
In my head "yield" makes more sense even though this is all based on killing states, but that's just documentation cosmetics for clarity perhaps. In that case we'd also have to prettify "Kill states when down" as while this is true the actual intention is to push them to another gateway immediately.

Reply:
we can fiddle with the wording before merge, I'm not sure about it either.